dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -60,43 +100,6 @@
 import edu.harvard.iq.dataverse.util.FileUtil;
 import io.vavr.control.Option;
 import io.vavr.control.Try;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang3.StringUtils;
-
-import javax.ejb.EJB;
-import javax.ejb.Stateless;
-import javax.inject.Inject;
-import javax.json.Json;
-import javax.json.JsonArrayBuilder;
-import javax.json.JsonObject;
-import javax.json.JsonObjectBuilder;
-import javax.json.JsonReader;
-import javax.persistence.Query;
-import javax.validation.ConstraintViolation;
-import javax.validation.ConstraintViolationException;
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.Response;
-import javax.ws.rs.core.Response.Status;
-import java.io.InputStream;
-import java.io.StringReader;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-import static edu.harvard.iq.dataverse.common.NullSafeJsonBuilder.jsonObjectBuilder;
-import static edu.harvard.iq.dataverse.util.json.JsonPrinter.json;
-import static edu.harvard.iq.dataverse.util.json.JsonPrinter.rolesToJson;
-import static edu.harvard.iq.dataverse.util.json.JsonPrinter.toJsonArray;
```

Comment:
Can you set your ide to organize imports: first group: all non static, next group: javax.* and java.*, last group: static imports? If we continue to mixing up organize order then we will have a lot of changes only for imports organizing